rulebook.tex

```diff
@@ -250,83 +278,103 @@ \section{Introduction}
  overall game activity while posing an additional strategic challenge to the
- teams\footnote{A list of rule changes can be found at
+ teams.
+ In 2020 the \ac{TC} reworked the storage station integration and the rules
+ regarding storage station usage. The storage station became fully functional.
+ In 2021 an online format was introduced to allow competitions around the
+ world without being required to gather at the same location or even having
+ a full field setup.
+ The online format consists of different challenges with varying difficulty.
+ While some of the complexity of the regular \ac{RCLL} game could not be
+ projected to the challenge format,
+ it allowed to introduce new ideas to the league,
+ that can be tested in isolated tasks before the final integration into
+ the regular game rules.
+ New teams also benefit from the challenge format as they can explore the
+ different aspects of the \ac{RCLL} without having to worry about the full scope
+ yet\footnote{A list of rule changes can be found at
  \mbox{\url{https://github.com/robocup-logistics/rcll-rulebook}}}.
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{The Task}
 \label{sec:task}
-
-Our aim is to provide a simplified Smart Factory environment. The
-teams must complete the following task without human interference,
-competing with a second team against the clock. In the RCLL,
-autonomous robot agents have to handle the logistics of materials
-through several (dynamic) stages to produce final goods to fulfill
-orders. The machines are specific MPS stations.
-
-The Logistics League's main challenge is a multistage production cycle
-of different product variants with self-crafted intermediate products
-and delivery of final products. This genuine goal will be rewarded
-considerably higher than partial fulfillment of the task. Autonomous
-robots transport small products between the processing
-machines. Machines are MPS stations which complete a particular
-refinement step like mounting a colored ring or top-most cap. If
-procurable this will lead to a new sub-assembly or final product
-(cap). Complete work orders require all related sub-assemblies of
-product variants (cf. \refsec{sec:production-complexities}).
-
-This work flow is controlled by a referee box broadcasting information
-via wifi (see \refsec{sec:referee-box}). The work flow itself is
-divided into three different phases: a setup
-phase (see \refsec{sec:setup-phase}), an exploration phase (see
-\refsec{sec:exploration-phase}) during which robots receive scores for
-correctly discovering and publishing the positions of the yet
-unknown different machines on the field. After this phase the referee
-box will announce all machine types and designations. In the following
-production phase (see~\refsec{sec:production-phase}) orders are
-announced by the referee box, which the robots must fulfill
+ Our aim is to provide a simplified Smart Factory environment, where
```

Comment:
Maybe also mention against each other? I really like that we have a direct competition, not like in the rescue where you only play against the clock.

Reply:
Ah just continue reading, and noticed that at the end there is the info that there are two tracks. In my opinion the text might be more easy to read if it is first mentioned that there are two tracks, and then borth are quickly summerized.

Reply:
Fixed in https://github.com/robocup-logistics/rcll-rulebook/pull/43/commits/c377ee9b79abb2f0415c18a518e7a8132b5c8a74